http_protocol_proposal_v1.md

```diff
@@ -62,7 +62,7 @@ Implementation MUST provide read access to `Correlation-Context` for logging sys
 
```

Comment:
Thanks!

I think our guidance here is a bit different:
1. First service should define correlation context.
2. Next service(s) MUST propagate it further without modifying properties, and SHOULD NOT add other properties. 

So the Correlation-Context is indeed read only for every service except the first one, but it is not a strict requirement.

Implementation does not really know which service is the first one so it must allow to add properties anyway.

I agree that I should be more clear about it in the spec. BTW, there is an [open PR](https://github.com/dotnet/corefx/pull/18887) to move this protocol to .net core repo. I'll do your changes there and also improve description. 
Once PR is merged, I'll change this doc to refer to the corefx repo.